keylime/tenant.py

```diff
@@ -640,6 +643,8 @@ def do_cvlist(self):
 
         response = do_cvstatus.get(f"/v{self.api_version}/agents/?verifier={verifier_id}", timeout=self.request_timeout)
 
+        Tenant._jsonify_response(response, True)
```

Comment:
Why not `response_json = Tenant._jsonify_response(response)` here as well and then also convert lines 654 and 657, 665, 669,  etc.

Reply:
same, thank you